.env.template.production

```diff
@@ -0,0 +1,29 @@
+## GENERAL ##
+# When running through Docker, always use: NODE_ENV=production
+NODE_ENV=production
+
+# The fully qualified domain name for the host where you are running Praise
+# Use HOST=localhost for development
+HOST=localhost
+
+# When running in development, API is accessed on this port, default: localhost:8088
+# In production, this port is used internally only since api is accessed
+# on $HOST/api 
+API_PORT=8088
+
+# Full URL to the host (and optionally port) where the API is running
+SERVER_URL=https://localhost
+
+# Full URL to the host (and optionally port) where frontend is being served
+FRONTEND_URL=https://localhost
+
+## DATABASE ##
+# Running through Docker: MONGO_HOST=mongodb
+# Running outside Docker: MONGO_HOST=localhost
+MONGO_HOST=mongodb
+MONGO_DB=praise_db 
+MONGO_PORT=27017
+MONGO_INITDB_ROOT_USERNAME=praiseDbRootUsername
+MONGO_INITDB_ROOT_PASSWORD=[any password]
+MONGO_USERNAME=praiseDbUsername
+MONGO_PASSWORD=[any password]
```

Comment:
Do you think it's necessary to have both `.env.template.production` and `.env.template.development`? IMO the comments make clear how to configure the env file so both templates are not necessary.

Reply:
It definitely saves some time when switching from dev to production and vice versa.

Reply:
We shouldn't need to switch from dev to production.

Once you setup a dev or production deployment -- you don't have to touch the env files except extremely rarely.